src/test/runtimeChoiceSelector.spec.ts

```diff
@@ -0,0 +1,102 @@
+import * as sinon from "sinon";
+import { expect } from "chai";
+import * as chai from "chai";
+import * as utils from "../utils";
+import * as runtime from "../runtimeChoiceSelector";
+import * as FirebaseError from "../error";
+// have to require this because no @types/cjson available
+// tslint:disable-next-line
+const cjson = require("cjson");
+
+describe("getRuntimeName", () => {
+  it("should properly convert raw runtime to human friendly runtime", () => {
+    expect(runtime.getHumanFriendlyRuntimeName("nodejs6")).to.contain("Node.js");
+  });
+});
+
+describe("getRuntimeChoice", () => {
+  const sandbox: sinon.SinonSandbox = sinon.createSandbox();
```

Comment:
```suggestion
  const sandbox = sinon.createSandbox();
```

Pretty sure it's redundant to declare the type inline like that :)

Reply:
Oh, I thought this was part of Typescript, like, defining the type of the constant. If it redundant in all cases or just this in particular?

Reply:
I don't think it's necessary. (a) you can't re-assign a `const` anyway, and (b) if it's a `let`, the compiler will shout if you've given it a type once and then try to change it.

[typescript playground](https://www.typescriptlang.org/play/#src=let%20n%3A%20number%3B%0D%0A%0D%0An%20%3D%203%3B%0D%0A%0D%0An%20%3D%20%224%22%3B%0D%0A%0D%0Aconst%20s%20%3D%20%224%22%0D%0A%0D%0As%20%3D%20%22asdf%22%3B%0D%0A%0D%0Alet%20v%20%3D%204%3B%0D%0A%0D%0Av%20%3D%20%22foo%22)

Reply:
Done.